src/main.rs

```diff
@@ -153,16 +153,30 @@ enum Sevctl {
     },
 }
 
-fn download(
-    rsp: reqwest::Result<reqwest::blocking::Response>,
-    usage: Usage,
-) -> Result<sev::Certificate> {
-    let rsp = rsp.context(format!("unable to contact {} server", usage))?;
-
-    let status = rsp.status();
-    let mut rsp = rsp.error_for_status().context(&format!(
-        "received failure from {} server: {}",
-        usage, status
+use std::time::Duration;
+
+fn download(url: &str, usage: Usage) -> Result<sev::Certificate> {
+    let mut rsp = reqwest::blocking::get(url);
+    for request_wait_seconds in &[0, 2, 4, 6, 9] {
+        std::thread::sleep(Duration::from_secs(*request_wait_seconds));
+        match &rsp {
+            // Connection is established, ensure that status code is not an error.
+            Ok(found) => {
+                if found.status().is_success() {
+                    break;
+                } else {
+                    rsp = reqwest::blocking::get(url);
+                }
+            }
+            // Connection not established, retry.
+            Err(_) => {
+                rsp = reqwest::blocking::get(url);
+            }
```

Comment:
When reading the documentation for [`reqwest::blocking::get`](https://docs.rs/reqwest/0.11.3/reqwest/blocking/fn.get.html), it lists the following as possible reasons why it will return `Err`:

* native TLS backend cannot be initialized
* supplied Url cannot be parsed
* there was an error while sending request
* redirect loop was detected
* redirect limit was exhausted

These don't seem like problems that will solve themselves if we retry. I would argue we probably aren't interested in retrying here. We should just propagate the error down.